promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,67 @@
+# Visualise Promscale traces in Jaeger
+This section shows you how to integrate
+[Jaeger][jaeger-ui] with Promscale.
+
+To visualize traces in Jaeger with Promscale, you need to have Jaeger query component running that queries and visualizes traces from Promscale. 
+
+<procedure>
+
+## Connect Jaeger query with Promscale
+
+1.  Deploy the Jaeger Query component from the Jaeger 
+    [deployments page][jaeger-deployments], if you aren't already running the Jaeger query.
+    
+    **Note**: For Jaeger to visualize traces in Promscale. We need Jaeger Query of version `1.30` or above. 
+    
+1.  Set the Jaeger query configuration through environment variables 
```

Comment:
Can we split this up to make it clearer?

Reply:
The action described in this step is setting up the environment variable. I do not like/prefer breaking the same action into multiple steps.